constants.go

```diff
@@ -43,12 +46,24 @@ const (
 	// PodNameFlag is a name of the flag of a pod name.
 	PodNameFlag = "pod-name"
 
+	// PodNamespaceEnvName is a name of the environment variable of a pod namespace.
+	PodNamespaceEnvName = "MYSQL_POD_NAMESPACE"
```

Comment:
There is a convention to name this as `POD_NAMESPACE`.
See https://github.com/kubernetes/kubernetes/blob/323f34858de18b862d43c40b2cced65ad8e24052/staging/src/k8s.io/client-go/tools/clientcmd/client_config.go#L527